src/libs/helpers.js

```diff
@@ -0,0 +1,10 @@
+import { CONFIG } from "./constants.js";
+import { getRandomNumberInRage } from "./utils.js";
+
+export function isMoveForward() {
+  const randomNumber = getRandomNumberInRage(CONFIG.MIN_RANDOM_NUMBER, CONFIG.MAX_RANDOM_NUMBER);
+  if (randomNumber >= CONFIG.RACER_MOVING_CONDITION) {
+    return true;
+  }
+  return false;
```

Comment:
```suggestion
  return randomNumber >= CONFIG.RACER_MOVING_CONDITION
```
이렇게 하면 더 간략하게 return 할 수 있을 것 같아요 👍